keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -56,6 +56,22 @@ Consider the following scenario:
 
 In this case, the administrator would like the evicted workload to be requeued as soon as possible on the newly available capacity.
 
+#### Story 2
+
+In the story 1 scenario, when we set `waitForPodsReady.requeuingTimestamp=Eviction`, 
+the workload endlessly or repeatedly can be put in front of the queue after eviction in the following eviction reasons:
+
+1. The workload don't have the proper configurations like image pull credential and pvc name, etc.
+2. The cluster can meet flavorQuotas, but each node doesn't have the resources that each podSet requests.  
```

Comment:
3. If there are multiple resource flavors that match the workload (flavors 1 & 2) and the workload is running on flavor 2 (where there happens to be a quota issue - i.e. flavor 2 is backed by on-demand capacity): there is a high likelihood that the workload will be requeued on the same flavor - with the same capacity issue (because other workloads will have taken up resource flavor 1 while the requeued workload was running.

Reply:
@nstogner Sorry, I couldn't understand when eviction happens, who is the preemptor, and who is the evictor.

Could you clarify?

Reply:
@nstogner case is still about waitForPodsReady timeout. Once the workload is evicted because it didn't become ready, there is the possibility that it won't fit in the first flavor, so it will be assigned back to the second flavor.

Reply:
Yes, my comment was in regards to the situation where eviction was due to waitForPodsReady. If the requeuing timestamp is configured to be the creation timestamp, and all resource flavors are in use, without backoff there is a very high likelihood that the freshly evicted workload will be readmitted into the same resource flavor, resulting in flapping.

Reply:
>  case is still about waitForPodsReady timeout. Once the workload is evicted because it didn't become ready, there is the possibility that it won't fit in the first flavor, so it will be assigned back to the second flavor.

Oh, it makes sense. I will add this story here.
@alculquicondor @nstogner Thank you.

Reply:
Done.